src/textual/constants.py

```diff
@@ -4,14 +4,31 @@
 
 from __future__ import annotations
 
+import enum
 import os
 
 from typing_extensions import Final
 
 get_environ = os.environ.get
 
 
-def get_environ_bool(name: str) -> bool:
+class AnimationsEnum(enum.Enum):
```

Comment:
We generally haven't used Enums in public interfaces, because they require an additional import step. And they are a little verbose. This also doesn't feel like the right place for this. I wouldn't think of importing `AnimationsEnum` from textual.contants.

Suggest we use a typing Literal, which is more typical for this codebase, and doesn't require an additional import. A literal gives us most of the benefits of eums.

Reply:
Sure thing.